telehealth-example/src/main/java/com/pubnub/components/example/telehealth/navigation/NavGraph.kt

```diff
@@ -0,0 +1,84 @@
+package com.pubnub.components.example.telehealth.navigation
+
+import android.net.Uri
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import com.google.gson.Gson
+import com.pubnub.api.PNConfiguration
+import com.pubnub.api.PubNub
+import com.pubnub.api.UserId
+import com.pubnub.api.enums.PNLogVerbosity
+import com.pubnub.components.example.telehealth.ChatApplication
+import com.pubnub.components.example.telehealth.dto.ChatParameters
+import com.pubnub.components.example.telehealth.dto.UserParameters
+import com.pubnub.components.example.telehealth.extensions.parcelable
+import com.pubnub.components.example.telehealth.ui.theme.AppTheme
+import com.pubnub.components.example.telehealth.ui.view.Channel
+import com.pubnub.components.example.telehealth.ui.view.Chat
+import com.pubnub.components.example.telehealth.ui.view.Login
+import com.pubnub.components.example.telehealth_example.BuildConfig
+
+@Composable
+fun NavGraph(navController: NavHostController) {
+    NavHost(
+        navController = navController,
+        startDestination = Screens.Login.route)
```

Comment:
To consider: Do we want to start with the Login Screen?

I understand that it's just a sample, but the correct behavior should be to move to the channel list.
See:
https://www.ittybittyapps.com/blog/2022-10-19-navigation-compose/
https://developer.android.com/guide/navigation/navigation-conditional